### ISSUE TITLE
Run "logging".   Get the following error.   CDK v2 OSE 3.2

### ISSUE THREAD

**Issue by alberttwong** [fabric8io/gofabric8#57]
When you run the template.  I get these errors.

![screen shot 2016-05-13 at 7 35 41 pm](https://cloud.githubusercontent.com/assets/749093/15265749/1264aba4-1942-11e6-9bb0-56949f0cb499.png)

**Comment by alberttwong** [comment]
Just for a double check... you run "management" or "logging" or both?

**Comment by jstrachan** [comment]
management = logging + metrics.

I wonder if this error is related to there being logging already installed when you try to install management? We should tidy it up a bit mind you to make it less surprising.

BTW even with this apparent failure; there should be services for kibana and elasticsearch in the `Services` tab right?

**Comment by alberttwong** [comment]
Yes I see the services but I don't know if they're working correctly.

**Comment by jstrachan** [comment]
@alberttwong ah cool. To see if they are working, you should be able to click on the Runtime -> Services tab and click on the URL for the kibana service and that should open a new tab to view the Kibana UI. 

I'm hoping this issue is fixed in a new release btw...
